src/main/java/org/jfrog/hudson/generic/ArtifactoryGenericConfigurator.java

```diff
@@ -204,6 +220,14 @@ public boolean isIncludeEnvVars() {
         return includeEnvVars;
     }
 
+    public boolean isDeployArtifacts() {
+        return deployArtifacts;
+    }
+
+        public IncludesExcludes getArtifactDeploymentPatterns() {
+        return artifactDeploymentPatterns;
+    }
+
```

Comment:
Why were the above methods added?

Reply:
No reasonable explanation. I was just trying to make the implementation work/compile and my two existing models / examples: ArtifactoryGradleConfigurator.java and ArtifactoryMaven3Configurator.java used that, so I probably took more that was need to resolve my compilation errors.

Reply:
just removed that.